frontend/src/components/Document/MyDocuments.js

```diff
@@ -1,15 +1,52 @@
-import { useParams } from "react-router-dom";
-import DocumentUpload from "../DocumentUpload/DocumentUpload";
+import { useEffect } from "react";
+import { useParams, NavLink } from "react-router-dom";
+import NewDocModal from "../Modals/NewDocModal";
+import DocumentUpload from "../DocumentPageTest/DocumentPageTest";
+import { useGetOneFolderMutation } from "../../store/features/api";
+import { useSelector } from "react-redux";
+import { Box, Container, Grid, GridItem } from "@chakra-ui/react";
+import { Link as ReactRouterLink } from "react-router-dom";
+import { Link as ChakraLink } from "@chakra-ui/react";
+import Document from "../Documents";
+import NewDocButton from "./newDocumentBtn";
+import ModalButton from "../Modals/ModalButton";
 
 // Display all documents related to a specific folder
 function MyDocuments() {
-    const { folderId } = useParams();
-    return(
-        <>
-            <div>This is folder number: {folderId}</div>
-            <DocumentUpload />
-        </>
-    )
+  const [getOneFolder] = useGetOneFolderMutation();
+  const { folderId } = useParams();
+  const folder = useSelector((state) => state.folder[folderId]);
+  const documents = folder?.documents;
+
```

Comment:
You shouldn't need the useEffect here. When you provide the tags at the end point it should cause a re-render when the db is updated.